src/components/login/Login.tsx

```diff
@@ -1,15 +1,36 @@
+// import { useState } from 'react'
+
+import { FormEvent, useState } from 'react'
+
+import { useNavigate } from 'react-router-dom'
+
+import LoginCredentials from '../../interfaces/LoginCredentials'
+import { getLogin } from '../../services/auth.service'
+
 import './Login.css'
 
-function Login() {
+function Login(props: { setIsLogged : React.Dispatch<React.SetStateAction<boolean>> }) {
+  const [ user, setUser ] = useState({ email: '', password: '' })
+  const navigate = useNavigate()
+
+  const handleSubmit = (e : FormEvent) => {
+    e.preventDefault()
+    const request: LoginCredentials = {
+      email: user.email,
+      password: user.password
+    }
+    getLogin(request).then(response => {
+      localStorage.setItem('token', response.data.accessToken)
```

Comment:
Would it be better to have token service so you get / set tokens from there instead of accessing localStorage directly in components? Also, there you can have constants with token keys instead of hardcoding 'token' key each time.